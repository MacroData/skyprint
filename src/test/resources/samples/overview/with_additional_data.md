# The Simplest API
This is one of the simplest APIs written in the **API Blueprint**.

# Group Test

