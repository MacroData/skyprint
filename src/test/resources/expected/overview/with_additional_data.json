{
  "name": "The Simplest API",
  "description": "This is one of the simplest APIs written in the **API Blueprint**."
}